SmartDeviceLink/SDLLogManager.m

```diff
@@ -120,6 +132,17 @@ - (void)logBytes:(NSData *)data direction:(SDLLogBytesDirection)direction timest
     [self logWithLevel:SDLLogLevelVerbose timestamp:timestamp file:file functionName:functionName line:line queue:queueLabel message:message];
 }
 
+- (void)logAssertWithTimestamp:(NSDate *)timestamp file:(NSString *)file functionName:(NSString *)functionName line:(NSInteger)line queue:(NSString *)queueLabel formatMessage:(NSString *)message, ... {
+    va_list args;
+    va_start(args, message);
+    NSString *format = [[NSString alloc] initWithFormat:message arguments:args];
+    va_end(args);
+
+    [self logWithLevel:SDLLogLevelError timestamp:timestamp file:file functionName:functionName line:line queue:queueLabel message:format];
+
+    NSAssert(self.areAssertionsDisabled, @"SDL ASSERTION: %@. To disable these assertions, alter your `SDLLogConfiguration` and set `disableAssertions` to `YES`", format);
```

Comment:
Seems to be a consequence of the `getter` attribute on the property. I considered removing that `getter` attribute, but since the others in the class have it, I think it's important that it should remain consistent.